plugins/modules/s3_bucket_info.py

```diff
@@ -192,225 +208,290 @@
             BlockPublicAcls:
               description: BlockPublicAcls setting value.
               type: bool
-              sample: true
             BlockPublicPolicy:
               description: BlockPublicPolicy setting value.
               type: bool
-              sample: true
             IgnorePublicAcls:
               description: IgnorePublicAcls setting value.
               type: bool
-              sample: true
             RestrictPublicBuckets:
               description: RestrictPublicBuckets setting value.
               type: bool
-              sample: true
     bucket_name_filter:
       description: String used to limit buckets. See I(name_filter).
-      returned: when I(name_filter) is defined
+      returned: when O(name_filter) is defined
       type: str
-      sample: filter-by-this-string
+      sample: "filter-by-this-string"
     bucket_acl:
       description: Bucket ACL configuration.
-      returned: when I(bucket_facts=true) and I(bucket_acl=true)
+      returned: when O(bucket_facts=true) and RV(buckets.bucket_acl=true).
       type: complex
       contains:
         Grants:
           description: List of ACL grants.
           type: list
-          sample: []
+          elements: dict
+          sample: [
+                    {
+                        "Grantee": {
+                            "DisplayName": "abc-aws-root+721234567890",
+                            "ID": "1234567890cbc0f1234567890f1234567890c123456789012123456789091234",
+                            "Type": "CanonicalUser"
+                        },
+                        "Permission": "FULL_CONTROL"
+                    }
+                  ]
+          contains:
+            Grantee:
+              description: The person being granted permissions.
+              type: dict
+              contains:
+                DisplayName:
+                  description: Screen name of the grantee.
+                  type: str
+                ID:
+                  description: The canonical user ID of the grantee.
+                  type: str
+                Type:
+                  description: Type of grantee.
+                  type: str
+        Permission:
+          description: Specifies the permission given to the grantee.
+          type: str
         Owner:
```

Comment:
Should we add a `returned` value to this?

Reply:
I guess we can leave it as `Permission`'s parent `bucket_acl` has a returned value of `returned: when O(bucket_facts=true) and RV(buckets.bucket_acl=true).`

Reply:
I think we can leave it out as the parent element `bucket_acl` has returned value "returned: when O(bucket_facts=true) and RV(buckets.bucket_acl=true)."